express/parsers/structure.py

```diff
@@ -155,18 +155,6 @@ def basis(self):
             'coordinates': [{'id': i + 1, 'value': v.frac_coords.tolist()} for i, v in enumerate(self.structure.sites)]
         }
 
-    def space_group_symbol(self):
```

Comment:
moved to crystal bc it is more specific to crystals than both crystals and molecules